newcomers.html

```diff
@@ -75,7 +75,8 @@
     <p>Each project has issues labeled "First Timers Only," try working on one of those for your first pull request.</p> 
     <p>Below there are tutorials and cheat sheets from Git to Glitch to Markdown. As well as information on how to help, even if you're not a developer.  Ask your questions on Slack, either on the #questions channel or in the project's channel (ex. #portal for the portal repo).</p>
 
-<div class="subheader"><h2>Helpful Sites & Tutorials</h2></div>
+
+<div class="subheader"><h3>Tutorials and Other Helpful Resources</h3></div>
```

Comment:
I don't mind changing the language on this header, but headers should be as concise as possible.  "Tutorials & Resources" would follow the style of other headers.  I prefer the ampersand instead of the word "and" written out for headers.